docs/reference/migration/migrate_5_0/scripting.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[breaking_50_scripting]]
+=== Script related changes
+
+==== Indexed scripts and templates
+
+Indexed scrips and templates have been replaced by <<modules-scripting-stored-scripts,stored scripts>>
+which stores the scripts and templates in the cluster state instead of a dedicate `.scripts` index.
+
+Previously indexed scripts in the `.scripts` index will not be used any more as
+Elasticsearch will now try to fetch the scripts from the cluster state. Upon upgrading
+to 5.x the `.scripts` index will remain to exist, so it can be used by a script to migrate
+the stored scripts from the `.scripts` index into the cluster state. The format of the scripts
+hasn't changed.
+
+The following Python script can be used to import your indexed scripts into the cluster state
+as stored scripts:
+
+[source,python]
+-----------------------------------
+from elasticsearch import Elasticsearch,helpers
+
+es = Elasticsearch([
+	{'host': 'localhost'}
+])
+
+for doc in helpers.scan(es, index=".scripts", preserve_order=True):
+	es.put_script(lang=doc['_type'], id=doc['_id'], body=doc['_source'])
+-----------------------------------
```

Comment:
@HonzaKral Would you like to verify if this script is okay? I did need to add `preserve_order=True`, otherwise the scan helper would fail, because it uses `search_type=scan` which has been removed in 5.0.